pkg/routeagent_driver/handlers/kubeproxy/vxlan.go

```diff
@@ -20,12 +20,12 @@ package kubeproxy
 
 import (
 	"errors"
-	"fmt"
 	"net"
 	"strconv"
 	"strings"
 	"syscall"
 
+	extErrors "github.com/pkg/errors"
```

Comment:
I think it would be clearer long term if we come up with a consistent set of package shorthands: _e.g._ `errors` for Golang `errors`, `exterrors` for the GH package, and `apierrors` for the K8s package.

Reply:
I use `apierrors` (or `apiErrors`) anyway. Re: the other two, we've mostly used the GH package so `errors` made sense. But with the newer APIs (eg `Is`, `As` etc) in the Golang package, we may be using that more. However I think GH will still be used much more so perhaps we should keep using `errors` for that and use `goerrors` (or `goErrors`) for the other.

Either way I think that should be a separate PR.

Reply:
Yeah, I was listing `apierrors` for completeness. `errors` for the GH package makes sense, with `goerrors` for the Golang package. Let’s make it a separate PR, yes.